changes.xml

```diff
@@ -23,6 +23,12 @@
     xsi:schemaLocation="http://maven.apache.org/changes/1.0.0 http://maven.apache.org/plugins/maven-changes-plugin/xsd/changes-1.0.0.xsd">
   <body>
 
+    <release version="2.1.0" date="not released">
```

Comment:
next release would be 2.0.2 as this is only a fix and not a new feature

Reply:
@stefanseifert updated to 2.0.2